topics/community/tutorials/community-tool-table/tutorial.md

```diff
@@ -4,6 +4,7 @@ title: Creation of an interactive Galaxy tools table for your community
 level: Introductory
 redirect_from:
 - /topics/dev/tutorials/community-tool-table/tutorial
+subtopic: sig
```

Comment:
in the metadata.yaml file you use `sigs` (with s) but in the tutorials without

Reply:
Gosh you're good